File: Node/src/dto/objects/Professional.ts
import { User } from "./User"

export namespace Professional {

    export namespace Parameters {
        
        export class Create {
            constructor() {}
        }
    }

    export namespace Responses {
        
        export class Partial {
            user: User.Responses.Partial
            
            constructor(
                user: User.Responses.Partial
            ) {
                this.user = user
            }
        }
    }
}

File: IOS/DTOs/Objects/Professional.swift
import Foundation
import MemberwiseInit

public enum Professional {
    public enum Parameters {}
    public enum Responses {}
}

public extension Professional.Parameters {
    
    @MemberwiseInit(.public, _optionalsDefaultNil: true)
    struct Create: Parametable {}
}

public extension Professional.Responses {
    
    @MemberwiseInit(.public, _optionalsDefaultNil: true)
    struct Partial: Responsable {
        public var user: User.Responses.Partial
    }
}
